docs/development/development.md

```diff
@@ -1,76 +1,96 @@
 # Developer Guide
-This document will walk you through how to set up a local development environment, build Akri component containers, and test Akri using your newly built containers.
+This document will walk you through how to set up a local development environment, build Akri component containers, and test Akri using your newly built containers. It also includes instructions on running Akri locally, naming guidelines, and points to our documentation on extending Akri with new Discovery Handlers and brokers.
```

Comment:
nit - remove 'our'. It is a good practice to avoid pronouns in technical writing.

Reply:
removed it! Thanks